crates/unftp-sbe-fs/examples/cap-ftpd.rs

```diff
@@ -0,0 +1,29 @@
+//! A server that jails each connected session with Capsicum.
+use std::{ffi::OsString, path::Path, str::FromStr};
+
+use unftp_sbe_fs::ServerExt;
+
+#[tokio::main(flavor = "current_thread")]
```

Comment:
Is it necessary to make it single threaded? I can imagine that we might want to keep the ability to offload something to another thread (for instance computationally heavy functions)

Reply:
The cap-ftpd.rs example doesn't do very much.  It just listens for new connections and then forks the helper process.  So there isn't much point to making it multi-threaded.  The cap-ftpd-worker could be multithreaded, too.  But since it only ever services one connection I think the benefit is also small.